README.rst

```diff
@@ -49,8 +49,6 @@ Grid fitters
 ------------
 
 - ``GridFit``: Abstract base class for grid fitters
-- ``Delaunay2DGridFit``: Interpolate scalar function defined on 2-D grid,
-  based on Delaunay triangulation
```

Comment:
Should the README maybe have a note about backwards-incompatible changes, just in case someone is actually using it.